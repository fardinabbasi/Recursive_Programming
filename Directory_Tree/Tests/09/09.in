7 5
file1 0
folder1 0 
file2 2 
file3 2 
folder2 2 
file4 5  
folder3 2