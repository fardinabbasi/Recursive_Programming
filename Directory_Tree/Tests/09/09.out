|__file1
|__folder1
   |__file2
   |__file3
   |__folder2
   |  |__file4
   |__folder3